Rg.Plugins.Popup/Rg.Plugins.Popup.csproj

```diff
@@ -31,6 +31,15 @@
     <Compile Remove="Resources\*.cs" />
     <Compile Update="**\*.xaml.cs" DependentUpon="%(Filename)" />
     <None Include="readme.txt" pack="true" PackagePath="." />
+    <Compile Update="Exceptions\InvalidPageException.cs">
```

Comment:
These changes should not be included. We use the wildcards.

Reply:
Ah, must've been added automatically by vs4mac, ill update that